README.md

```diff
@@ -1,234 +1,262 @@
 # truckersmp-cli
 
-truckersmp-cli is an easy to use script to download TruckersMP and start
-the game afterwards.
+A simple launcher for TruckersMP to be used with Proton or Wine.
 
-It can install and update the windows version of
-American Truck Simulator (`-a`, `--ats`) or Euro Truck Simulator 2 (`-e`, `--ets2`)
-with steamcmd (`-u`, `--update`) and handles starting (`-s`, `--start`) the mod
-through Proton aka. Steam Play (`-p`, `--proton`) or Wine (`-w`, `--wine`).
+It can install and update the Windows version of American Truck Simulator and Euro Truck Simulator 2 with steamcmd and handles starting the multiplayer mod through Proton or Wine.
+A working native Steam installation is needed for starting through Proton - when using standard Wine you should have the Windows version of Steam already running.
+While updating the launcher will stop all running Steam processes to prevent Steam from asking for password and guard code at the next startup.
```

Comment:
Proton and (Linux version of) steamcmd only support Linux systems.

Could you take other OSes into consideration? We need "on Linux" or something else for Linux-only stuff.